kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -20,7 +20,6 @@ window.ContentAreaView = BaseView.extend({
     show_view: function(view) {
 
         // hide any messages being shown for the old view
```

Comment:
What is the rationale for not clearing messages when we show a new content view?